boom! testing num-derive-0.1.40 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! 22344fa05d15a463bce115460be3c23695a168f36dcd2e07e82d789a48cc8338
boom! running `docker start -a 22344fa05d15a463bce115460be3c23695a168f36dcd2e07e82d789a48cc8338`
kablam!    Compiling syn v0.7.1
kablam!    Compiling num-derive v0.1.40 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 7.3 secs
boom! running `docker rm -f 22344fa05d15a463bce115460be3c23695a168f36dcd2e07e82d789a48cc8338`
blam! 22344fa05d15a463bce115460be3c23695a168f36dcd2e07e82d789a48cc8338
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! 636cb3a28fdff7551b20d3e111de086e3d5dd69afa501b685ad39e3ee3fa6f3e
boom! running `docker start -a 636cb3a28fdff7551b20d3e111de086e3d5dd69afa501b685ad39e3ee3fa6f3e`
kablam!    Compiling num-derive v0.1.40 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 2.35 secs
boom! running `docker rm -f 636cb3a28fdff7551b20d3e111de086e3d5dd69afa501b685ad39e3ee3fa6f3e`
blam! 636cb3a28fdff7551b20d3e111de086e3d5dd69afa501b685ad39e3ee3fa6f3e
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! 6091aca5422b5e0e965a96ff1e93413480d8ff9398e2267c3e89d3884ecd9c59
boom! running `docker start -a 6091aca5422b5e0e965a96ff1e93413480d8ff9398e2267c3e89d3884ecd9c59`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/compiletest-9cff2e95fb9f331b
blam! 
blam! running 1 test
blam! 
blam! running 2 tests
blam! test [compile-fail] compile-fail/enum_with_associated_data.rs ... FAILED
blam! test [compile-fail] compile-fail/derive_on_struct.rs ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- [compile-fail] compile-fail/enum_with_associated_data.rs stdout ----
blam! 	
blam! error: tests/compile-fail/enum_with_associated_data.rs:11: unexpected "error": '11:1: 11:18: can't find crate for `num` [E0463]'
blam! 
blam! error: tests/compile-fail/enum_with_associated_data.rs:15: expected error not found: 
blam! 
blam! error: 1 unexpected errors found, 1 expected errors not found
blam! status: exit code: 101
blam! command: rustc tests/compile-fail/enum_with_associated_data.rs -L /tmp --target=x86_64-unknown-linux-gnu --error-format json -L /tmp/enum_with_associated_data.stage-id.compile-fail.libaux -C prefer-dynamic -o /tmp/enum_with_associated_data.stage-id -L target/debug/ -L target/debug/deps/
blam! unexpected errors (from JSON output): [
blam!     Error {
blam!         line_num: 11,
blam!         kind: Some(
blam!             Error
blam!         ),
blam!         msg: "11:1: 11:18: can\'t find crate for `num` [E0463]"
blam!     }
blam! ]
blam! 
blam! not found errors (from test file): [
blam!     Error {
blam!         line_num: 15,
blam!         kind: Some(
blam!             Error
blam!         ),
blam!         msg: ""
blam!     }
blam! ]
blam! 
blam! thread '[compile-fail] compile-fail/enum_with_associated_data.rs' panicked at 'explicit panic', /cargo-home/registry/src/github.com-1ecc6299db9ec823/compiletest_rs-0.2.5/src/runtest.rs:1097
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- [compile-fail] compile-fail/derive_on_struct.rs stdout ----
blam! 	
blam! error: tests/compile-fail/derive_on_struct.rs:11: unexpected "error": '11:1: 11:18: can't find crate for `num` [E0463]'
blam! 
blam! error: tests/compile-fail/derive_on_struct.rs:15: expected error not found: 
blam! 
blam! error: 1 unexpected errors found, 1 expected errors not found
blam! status: exit code: 101
blam! command: rustc tests/compile-fail/derive_on_struct.rs -L /tmp --target=x86_64-unknown-linux-gnu --error-format json -L /tmp/derive_on_struct.stage-id.compile-fail.libaux -C prefer-dynamic -o /tmp/derive_on_struct.stage-id -L target/debug/ -L target/debug/deps/
blam! unexpected errors (from JSON output): [
blam!     Error {
blam!         line_num: 11,
blam!         kind: Some(
blam!             Error
blam!         ),
blam!         msg: "11:1: 11:18: can\'t find crate for `num` [E0463]"
blam!     }
blam! ]
blam! 
blam! not found errors (from test file): [
blam!     Error {
blam!         line_num: 15,
blam!         kind: Some(
blam!             Error
blam!         ),
blam!         msg: ""
blam!     }
blam! ]
blam! 
blam! thread '[compile-fail] compile-fail/derive_on_struct.rs' panicked at 'explicit panic', /cargo-home/registry/src/github.com-1ecc6299db9ec823/compiletest_rs-0.2.5/src/runtest.rs:1097
blam! 
blam! 
blam! failures:
blam!     [compile-fail] compile-fail/derive_on_struct.rs
blam!     [compile-fail] compile-fail/enum_with_associated_data.rs
blam! 
blam! test result: FAILED. 0 passed; 2 failed; 0 ignored; 0 measured
blam! 
blam! test compile_test ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- compile_test stdout ----
blam! 	thread 'compile_test' panicked at 'Some tests failed', /cargo-home/registry/src/github.com-1ecc6299db9ec823/compiletest_rs-0.2.5/src/compiletest.rs:122
blam! 
blam! 
blam! failures:
blam!     compile_test
blam! 
blam! test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f 6091aca5422b5e0e965a96ff1e93413480d8ff9398e2267c3e89d3884ecd9c59`
blam! 6091aca5422b5e0e965a96ff1e93413480d8ff9398e2267c3e89d3884ecd9c59
